curriculum/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+class Chapter(models.Model):
+    number    = models.CharField(max_length=100)
+    title     = models.CharField(max_length=200)
+    product   = models.ForeignKey("product.Product",  on_delete=models.SET_NULL, null=True)
+    position  = models.IntegerField(null=True)
+
+    class Meta:
+        db_table = 'chapters'
+
+class Content(models.Model):
+    number   = models.CharField(max_length=10)
+    title    = models.CharField(max_length=100)
+    chapter  = models.ForeignKey('Chapter', on_delete=models.SET_NULL, null=True)
+    product  = models.ForeignKey('product.Product', on_delete=models.SET_NULL, null=True)
+    preview  = models.BooleanField(default=False)
+    position = models.IntegerField(null=True)
+
+    class Meta:
+        db_table = 'contents'
+
+class Comment_content(models.Model):
+    comment = models.TextField()
+    photo   = models.ImageField(upload_to="")
+    pin     = models.BooleanField(default=False)
+    user    = models.ForeignKey('user.User', on_delete=models.SET_NULL, null=True)
+    content = models.ForeignKey('Content', on_delete=models.SET_NULL, null=True)
+    created_at = models.DateTimeField(auto_now_add = True)
+    update_at  = models.DateTimeField(auto_now = True)
```

Comment:
```suggestion
class Comment(models.Model):
    comment = models.TextField()
    photo   = models.ImageField(upload_to="")
    pin     = models.BooleanField(default=False)
    user    = models.ForeignKey('user.User', on_delete=models.SET_NULL, null=True)
    content = models.ForeignKey('Content', on_delete=models.SET_NULL, null=True)
    created_at = models.DateTimeField(auto_now_add = True)
    update_at  = models.DateTimeField(auto_now = True)
```
클래스 이름에는 언더바를 사용하지 않는 것이 파이썬 규칙입니다. (CommentContent)
의미상 `CommetContent`라 규정하지 않고 그냥 `Comment`로 설정하셔도 좋을 것 같습니다 :)

Reply:
`=`기준으로 코드 간격 수정해주세요

Reply:
이름 변경하고 간격 수정했습니다.